Dockerfile.webmon

```diff
@@ -1,10 +1,10 @@
-FROM continuumio/miniconda3:4.12.0
+FROM ghcr.io/prefix-dev/pixi:0.50.2-jammy
```

Comment:
just out of curiosity, was there a reason for using a different base image for webmon?

Reply:
I had changed it at one point to try to get `update-ca-certificates` to run. I ended up needing to `apt install ca-certificates` anyway so it probably could be change back to match the others.